kuma/banners/models.py

```diff
@@ -20,7 +20,8 @@ class Banner(models.Model):
     title = models.CharField('Banner Title', max_length=100)
     main_copy = models.TextField('Main Copy', max_length=200)
     button_copy = models.CharField('Button Copy', max_length=50)
-    theme = models.CharField('Theme', max_length=20,
+    button_url = models.URLField('URL')
```

Comment:
This line doesn't match the migration. Where is the `verbose_name` and the `default`?

Reply:
Whoops, my bad for manually merging and editing the migration directly ;) Fixing.